ingestion/functions/README.md

```diff
@@ -226,6 +226,12 @@ If a source has a time-based URL scheme you can use the following date formattin
 
 For example if a source publishes its data every day at a URL like `https://source.com/data/year-month-day.json` you can set the source URL to `https://source.com/data/$FULLYEAR-$FULLMONTH-$FULLDAY.json` and it will fetch the URL `https://source.com/data/2020-04-20.json` on the 4th of April 2020.
 
+### Compressed sources
+
+Some sources are provided as [zip files](https://en.wikipedia.org/wiki/Zip_(file_format)). Those are supported by the retrieval function assuming it contains a single file in the archive containing the line list data, it will extract that single file and the parsing functions will have access to it so you can write a parser without caring about the zip file at all.
+
+If you need other archive or compression formats supported please file an issue in this repository indicating the source and the desired support, thank you.
```

Comment:
perhaps:

\[file an issue in this repository\]\(https://github.com/globaldothealth/list/issues/new?assignees=&labels=Importer&template=feature_request.md&title=Additional%20compression%20support\)

Reply:
thanks, that's nice. Done.